src/org/rascalmpl/interpreter/Evaluator.java

```diff
@@ -804,14 +810,12 @@ public ParserGenerator getParserGenerator() {
             }
 
             Evaluator self = this;
-            job("Loading parser generator", () -> {
```

Comment:
Yes now you can read this off of the time it took to load the ParserGenerator module. It will be printed right above the new prompt because ParserGenerator is the top module which will only be finished when all the modules it depends on have been loaded.